src/components/Auth/UsersCard/table/UsersTable.module.scss

```diff
@@ -1,13 +1,43 @@
+@import '../../../common/color';
+
 .zeroStateWrapper {
-  height: 200px;
+  height: 100px;
   display: flex;
+  color: $COLOR_TEXT_SECONDARY;
   justify-content: center;
   align-items: center;
 }
 
 .noResultsWrapper {
-  height: 50px;
+  height: 100px;
+  color: $COLOR_TEXT_SECONDARY;
   display: flex;
   justify-content: center;
   align-items: center;
 }
+
+// Double selector - for higher specificity to Override rmwc styles.
+.tableWrapper.tableWrapper {
+  // Avoid double borders.
+  margin: -1px;
+  border-radius: 0 0 4px 4px;
+}
+
+.disabled {
```

Comment:
Optional: Consider using a `<ThemeProvider>` around the table row in JSX. However if that takes too much effort, just bail on it and we can always get back to this later. (The other simple color overrides in this file needs to be fixed though.)

Reply:
Added a TODO for now